client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -1,5 +1,65 @@
-const ZoneCreator = () => {
-	return null;
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { FormSection, reduxForm } from 'redux-form';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+import FormLabel from 'components/forms/form-label';
+import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SectionHeader from 'components/section-header';
+
+const ZoneCreator = ( { translate } ) => {
+	return (
+		<div>
+			<form>
+				<FormSection name="newZone">
+					<SectionHeader label={ translate( 'New Zone' ) } />
+
+					<Card>
+						<p>
+							{ translate(
+								'To create a zone, enter a name (and any other info) and click "Add Zone". ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Add DNS Zone' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/zoninator-new-zone-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).